tvb_contrib/tvb/contrib/tests/cosimulation/parallel/double_proxy_precision_complex_delay_update_test.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+#
+#  TheVirtualBrain-Contributors Package. This package holds simulator extensions.
+#  See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+
+"""
+.. moduleauthor:: Lionel Kusch <lkusch@thevirtualbrain.org>
+.. moduleauthor:: Dionysios Perdikis <dionperd@gmail.com>
+"""
+
+import numpy as np
+import numpy.random as rgn
+
+from tvb.tests.library.base_testcase import BaseTestCase
+from tvb.contrib.tests.cosimulation.parallel.function_tvb import TvbSim
+
+
+class TestDoubleProxyPrecisionComplexDelayUpdate(BaseTestCase):
+    """
+    test the transmission of information between two models with proxy in most complex case and different delay
+    """
+    def test_double_precision_complex_delay_update(self):
+        weight = np.array([[5, 2, 4, 0], [8, 5, 4, 1], [6, 1, 7, 9], [10, 0, 5, 6]])
```

Comment:
NumPy will use integer weights here, probably not a big deal, but will affect testing precision below.  Use `]], np.float64)` at the end, for isntance.

Reply:
By looking the code, I find that the weight are define by np.float instead of np.float64 .
It's a bug or not?
I prefer to add np.float as the code of the connectivity

Reply:
```python
>>> np.float==float
True
```
I guess that means that the weights even if given as integers will be coerced to floats, which is what I meant so it's fine.